client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
```

Comment:
Since you only have one list item in this jsx, you probably don't need to use a `<ul>` and `<li>` here at all - you could remove them and just have all of the divs be direct children of the section.

Reply:
Yeah, I have corrected that. Thank you 👍